src/style.css

```diff
@@ -1,3 +1,64 @@
+:root {
+  --blue: #00acdc;
+}
+
 body {
+  font-family: Roboto, Helvetica, Arial, sans-serif;
+  margin: 0;
+  background-color: aqua;
+  color: var(--blue);
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  background-color: white;
+}
+
+.logo {
+  margin: auto auto auto 50px;
+}
+
+.nav {
+  margin: auto 50px auto auto ;
+  width: 100%;
+}
+
+.list {
+  display: flex;
+  justify-content: end;
   margin: 0;
 }
+
+.nav-item {
+  list-style-type: none;
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. In this case, you could add a class to the list items and style them using the class.